### PR TITLE
Check token to view the page.

### PR DESCRIPTION
**Pull request by md-jafari**

### CODE REVIEW

**Review by HenryOulai — APPROVED** [review]
good work

File: src/app/provider_page/page.tsx [hunk 1]
```diff
@@ -1,7 +1,57 @@
...
+'use client'
+import { useEffect, useState } from "react";
+import Logo from "../components/logo";
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { DateCalendar } from '@mui/x-date-pickers/DateCalendar';
+import { useRouter } from "next/navigation";
+import { UserRole } from "../utils/interface";
+
+export default function PatientPage() {
```
Comment (Bisi0n) [inline src/app/provider_page/page.tsx:10]: Just small syntax, change to ProviderPage

File: src/app/patient_page/page.tsx [hunk 2]
```diff
@@ -1,17 +1,29 @@
...
 import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider'
 import { DateCalendar } from '@mui/x-date-pickers/DateCalendar'
 
 export default function PatientPage() {
-
-
+    const [patients, setPatients] = useState<Patient[]>([])
+    
+    useEffect(() => {
+        fetch(`${process.env.NEXT_PUBLIC_API_URL}/api/login`) // TODO: create api folder and move the fetch there
```
Comment (Bisi0n) [inline src/app/patient_page/page.tsx:13]: change endpoint to user/current

File: src/app/utils/interface.tsx [hunk 3]
```diff
@@ -22,4 +22,12 @@ export interface Provider {
 
 export interface NavBarProps {
     currentPath: string
+}
+
+export interface Bearer {
```
Comment (Bisi0n) [inline src/app/utils/interface.tsx:27]: unnecessary interface since we're handling the token in backend for now

File: package.json [hunk 4]
```diff
@@ -14,12 +14,15 @@
     "@mui/material": "^5.15.3",
     "@mui/styled-engine-sc": "^6.0.0-alpha.12",
     "@mui/x-date-pickers": "^6.19.0",
+    "@types/js-cookie": "^3.0.6",
     "dayjs": "^1.11.10",
+    "js-cookie": "^3.0.5",
```
Comment (Bisi0n) [inline package.json:19]: Uninstall unused package by using "npm uninstall js-cookie"

**Review by Bisi0n — CHANGES_REQUESTED** [review]
Just minor issue

**Review by Bisi0n — APPROVED** [review]
Nice, Well done!

**Pull request merged**